2016/proposals/pravendra-singh.md

```diff
@@ -0,0 +1,220 @@
+# A survey responses visualizer application
+
+## Abstract
+
+Software Carpentry has been educating multidisciplinary researchers with the necessary computing skills to help them in their venture. It enables the same via workshops for different fields, community discussion between volunteers, and curated readings etc.
+
+The number of people attending the workshops itself is sufficient enough to help us imagine the impact Software Carpentry Foundation has been making since its foundation.
+
+![number-of-learners](http://software-carpentry.org/files/graphs/learners-2015-10.png)
+[credits: software-carpentry.org]
+
+Software Carpentry maintains a set of tools that help them manage their workshop events. For example, the project [AMY](https://github.com/numfocus/amy) is a workshop management tool for the organization.
+
+There are *pre* and *post* workshop surveys also. They support in the proper event organization by collecting details about expected attendees for *pre-workshop* surveys and feedback about the event for *post-workshop* surveys.
+
+The following proposal talks down an application which can "*Represent the survey responses in a comprehensible manner, assisting the workshop organizers (hosts) to plan the event according to the feedback from attendees*".
+
+The application will be a relieve from the current cumbersome method we use for data aggregation.
```

Comment:
Yes, it's a bit long because you included "introduction to Software Carpentry" – I don't mind it, though, and we certainly won't be grading the proposals based on the length of the abstract. I wouldn't worry about it ;)